src/core/broker_config.cpp

```diff
@@ -57,40 +227,65 @@ BrokerConfig::ParseResult BrokerConfig::ReadFromFile(const std::string& file_nam
   return ReadFromStream(file_stream);
```

Comment:
You're not closing the ifstream.

Reply:
Thanks! But, does it not get closed on destruction?

Reply:
Oops.  Yes it does.  cppreference is better than cplusplus at some things. :)